src/Lesson_2/Dz_5.java

```diff
@@ -0,0 +1,15 @@
+package Lesson_2;
+
+public class Dz_5 {
+    public static void main(String[] args) {
+        longYear(2023);
+    }
+    public static boolean longYear(int b) {
```

Comment:
```suggestion
    public static boolean isLongYear(int b) {
```
обычно для boolean методов используется данный префикс